hq/app/views/snyk/snyk.scala.html

```diff
@@ -0,0 +1,62 @@
+@import logic.SnykDisplay
+@import logic._
+@import model._
+
+@(results: List[SnykProjectIssues])(implicit assets: AssetsFinder)
+
+@main(Nil) { @* Header *@
+    <div class="container">
+        <div class="row">
+            <h1 class="header center-on-small-only grey-text text-lighten-5">Security HQ</h1>
+            <div class="col s12 m9">
+                <h4 class="grey-text text-darken-4 center-on-small-only">Snyk Results</h4>
+            </div>
+        </div>
+    </div>
+
+} { @* Main content *@
+
+    <div class="container">
+
+        <table class="striped responsive-table filterable-table">
+            <thead>
+                <tr>
+                    <th>Project Name</th>
+                    <th>High</th>
+                    <th>Medium</th>
+                    <th>Low</th>
+                </tr>
+            </thead>
+
+            <tbody>
+            @for(project <- results.sortWith((a, b) =>
+                a.high > b.high
+                        || a.high == b.high && (
+                        a.medium > b.medium
+                                || a.medium == b.medium && (
+                                a.low > b.low
+                                        || a.low == b.low && a.name < b.name)))) {
```

Comment:
Yay! Sorting 🎉 

Not a blocker, but could this logic be moved to a separate logic file over in Scala land?
 Maybe logic/SnykDisplay.scala?

Reply:
Done.